test/consume_test.py

```diff
@@ -80,6 +78,8 @@ def test_loop_device(user_loop):
 
 def test_mount(user_mount):
     assert os.path.isdir(user_mount.path)
+    if user_mount.fstype == "ext2":
+        pytest.xfail("block size detection broken since kernel 5.5")
```

Comment:
Why skip the entire test when the only thing we cannot test is line 88?

We can do:

    if user_mount.fstype == "ext2":
        log.warning(...)
    else:
        assert detect_block_size(filename) == user_mount.sector_size